.github/workflows/clean-ghcr.yaml

```
@@ -0,0 +1,30 @@
+---
+name: Delete Obsolete GHCR Images
+
+on:
+  workflow_call:
+    inputs:
+      image-names:
+        type: string
+        required: true
+        description: The names of the container images you want to delete old versions for.
```

Hva slags delimiter?
```suggestion
        description: The newlin-separated names of the container images you want to delete old versions for.
```

---

Updated